src/java.desktop/share/classes/java/awt/GridLayout.java

```diff
@@ -32,25 +32,36 @@
  * lays out a container's components in a rectangular grid.
  * The container is divided into equal-sized rectangles,
  * and one component is placed in each rectangle.
- * For example, the following is an applet that lays out six buttons
+ * For example, the following is a window that lays out six buttons
  * into three rows and two columns:
  *
- * <hr><blockquote>
+ * <hr>
+ * {@snippet lang='java':
  * <pre>
- * import java.awt.*;
- * import java.applet.Applet;
- * public class ButtonGrid extends Applet {
- *     public void init() {
- *         setLayout(new GridLayout(3,2));
- *         add(new Button("1"));
- *         add(new Button("2"));
- *         add(new Button("3"));
- *         add(new Button("4"));
- *         add(new Button("5"));
- *         add(new Button("6"));
- *     }
+ * import java.awt.Button;
+ * import java.awt.EventQueue;
+ * import java.awt.Frame;
+ * import java.awt.GridLayout;
+ *
+ * public class GridLayoutExample {
+ *
+ *   public static void main(String[] args) throws Exception {
+ *
+ *     EventQueue.invokeAndWait(() -> {
+ *        Frame frame = new Frame("GridLayout");
```

Comment:
Let's indent the code by 4 spaces.

I'm also for removing the blank line that starts the `main` method.